client/src/components/Explorer/Explorer.scss

```diff
@@ -1,5 +1,3 @@
-// scss-lint:disable DeclarationOrder
```

Comment:
A search for "order" on https://stylelint.io/user-guide/rules/ returned no results, but I haven't exhaustively read all the rules, so let me know if you see it!